packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -193,7 +230,7 @@ export type TAppLinks = {
 export const LinksMapper: Record<TPlatforms.All, TAppLinks> = {
     ctrader: {
         android: 'https://play.google.com/store/apps/details?id=com.deriv.ct',
-        ios: 'https://apps.apple.com/cy/app/ctrader/id767428811',
+        ios: 'https://apps.apple.com/us/app/deriv-ctrader/id6466996509',
```

Comment:
Did you try to remove this versioning to see if it's offering the latest version or not?

Reply:
I did, it does not redirect to the app if I remove the id